verify.rb

```diff
@@ -3,28 +3,28 @@
 require 'kwalify'
 @output = 0
 
-# YAML tags related to TFA
-@tfa_tags = {
-  # YAML tags for TFA Yes
-  true => %w[email hardware software sms phone doc],
-  # YAML tags for TFA No
-  false => %w[status twitter facebook email_address lang]
-}.freeze
-
 # Image max size (in bytes)
 @img_max_size = 2500
+#@img_max_size = 3000
 
 # Image dimensions
 @img_dimensions = [32, 32]
+@img_lg_dimensions = [48, 48]
```

Comment:
Same here, just curious why the bump. Don't think it will hurt anything repo-size wise.

Reply:
We plan to use 48x48 for charity page. The page is gonna be different so I added the size to validation